src/Handshake/ServerNegotiator.php

```diff
@@ -81,7 +99,7 @@ public function handshake(RequestInterface $request) {
             }, null);
 
             if ($this->_strictSubProtocols && null === $match) {
-                return new Response(400, [], '1.1', null ,'No Sec-WebSocket-Protocols requested supported');
+                return new Response(426, $upgradeSuggestion);
```

Comment:
I started looking into this change and got confused. The HTTP spec is vague but seems to say 426 is if the server doesn't like the "Upgrade" header...but 6455 says also use 426 if the server doesn't like "Sec-WebSocket-Version" -- so I feel like either 6455 is bending the rules of 426 or the HTTP spec could use better wording...either way this seems ok given the pattern.